sdcm/utils/adaptive_timeouts/__init__.py

```diff
@@ -48,6 +55,7 @@ class Operations(Enum):
     SCRUB = ("scrub", _get_soft_timeout, ("timeout",))
     SOFT_TIMEOUT = ("soft_timeout", _get_soft_timeout, ("timeout",))
     FLUSH = ("flush", _get_soft_timeout, ("timeout",))
+    QUERY = ("query", _get_query_timeout, ("timeout",))
```

Comment:
you missed "query" parameter - this causes ES to not get the value for it.
See https://70f106c98484448dbc4705050eb3f7e9.us-east-1.aws.found.io:9243/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(columns:!(),filters:!(),index:'94fb1d70-eb66-11ed-b96a-256b7e8b2261',interval:auto,query:(language:kuery,query:'operation%20:%20QUERY'),sort:!())

Reply:
fixed